test/selenium/specs/tc_new.rb

```diff
@@ -10,6 +10,7 @@ class TestNew < Test::Unit::TestCase
   def setup
     page.driver.browser.get(@@privly_test_set[0][:url])
     login(@@privly_test_set[0][:content_server])
+    page.driver.browser.manage.timeouts.implicit_wait = 3
```

Comment:
It is better to have an implicit wait to avoid NoSuchElementError but, the tests pass successfully even without having the implicit_wait.
Should this line be removed then?